### PR TITLE
Adding support for connecting standalone PEM in pxapi

### PR DESCRIPTION
**Pull request by RagalahariP**
Summary: Adding support for connecting standalone PEM in `pxapi`. 

Test Plan: Added an example program to connect standalone PEM

### CODE REVIEW

**Comment by RagalahariP** [comment]
To add, observed that `vizierID`  in `NewVizierClient()` is not getting used while extracting results from standalonePEM .  Please correct my observation.

**Review by aimichelle — CHANGES_REQUESTED** [review]

File: src/api/go/pxapi/client.go [hunk 1]
```diff
@@ -79,12 +80,34 @@ func NewClient(ctx context.Context, opts ...ClientOption) (*Client, error) {
...
+
 	if err := c.init(ctx); err != nil {
 		return nil, err
 	}
 	return c, nil
 }
 
+
+// initLocalClient is for establishing gRPC connection to standalonePEM
+func (c *Client) initLocalClient(ctx context.Context) error {
```
[the remark this answers is not in the document]
Comment (RagalahariP) [reply src/api/go/pxapi/client.go]: @aimichelle Thank you for review. Updated code as suggested.

File: src/api/go/pxapi/client.go [hunk 2]
```diff
@@ -79,12 +81,34 @@ func NewClient(ctx context.Context, opts ...ClientOption) (*Client, error) {
 		opt(c)
 	}
 
+	if (strings.Contains(c.directAddr, "0.0.0.0") == true) {
```
Comment (JamesMBartlett) [inline src/api/go/pxapi/client.go:84]: This check should be `c.directAddr != ""`

File: src/api/go/pxapi/client.go [hunk 3]
```diff
@@ -79,12 +81,34 @@ func NewClient(ctx context.Context, opts ...ClientOption) (*Client, error) {
...
 	if err := c.init(ctx); err != nil {
 		return nil, err
 	}
 	return c, nil
 }
 
+
+// initDirectClient is for establishing gRPC connection to standalonePEM
+func (c *Client) initDirectClient(ctx context.Context) error {
+        conn, err := grpc.Dial(c.directAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
```
Comment (JamesMBartlett) [inline src/api/go/pxapi/client.go:100]: I think we should take the credentials in as another `ClientOption` eg `WithDirectCreds`

**Comment by vihangm** [comment]
Looks like this was covered by #1762

**Pull request closed without merging**